include/mbedtls/config.h

```diff
@@ -84,7 +84,7 @@
  * example, if double-width division is implemented in software, disabling
  * it can reduce code size in some embedded targets.
  */
-//#define MBEDTLS_NO_UDBL_DIVISION
+#define MBEDTLS_NO_UDBL_DIVISION
```

Comment:
This, and numerous similar changes, seem unintentional.

Reply:
fixed.